src/weather-bar.ts

```diff
@@ -128,7 +133,7 @@ export class WeatherBar extends LitElement {
           <div class="bar-block-bottom">
             <div class="date">${renderedDate}</div>
             <div class="hour">${hideHours ? null : hour}</div>
-            <div class="temperature">${hideTemperature ? null : html`${temperature}&deg;`}</div>
+            <div class="temperature">${hideTemperature ? null : roundedTemperature !== null ? html`${roundedTemperature}&deg;` : html`${temperature}&deg;`}</div>
```

Comment:
Under what circumstance would `displayTemperature` be `null`?

```suggestion
            <div class="temperature">${hideTemperature ? null : displayTemperature !== null ? html`${displayTemperature}&deg;` : html`${temperature}&deg;`}</div>
```

Reply:
If round_temperatures was false then displayTemperature would be null.
[I've change this in commit 67e45b4](https://github.com/decompil3d/lovelace-hourly-weather/pull/609/commits/67e45b4178089513858ba3b530e5b4c72b30512a)

```typescript
<div class="temperature">${hideTemperature ? null : this.round_temperatures ? html`${displayTemperature}&deg;` : html`${temperature}&deg;`}</div>
```